docs/source/onboarding_resources/computing/index.rst

```diff
@@ -125,39 +125,88 @@ in the Accessing the Cluster section below.
 2.0 How to Access a Terminal
 ++++++++++++++++++++++++++++
 
-.. todo::
+A lot of work you will do requires you to access a terminal. A terminal is a way for you, a 
+user, to communicate with a computer or computing system. There are many options for 
+terminals. This section is written for someone not familiar with terminals, so if 
+you have familiarity or favorite software please feel free to use those! 
 
-  - Build out this section with terminal options, Linux vs Unix vs Windows 
-  - Link to the command line training here and explain that it is Linux based 
-  - The below section needs to be reworked a bit but was pasted as a starting point 
-  - Git comes with windows but is not Linux based 
-  - Seems that some terminals require Git tokens and others don't. Figure out which 
-  - How to install conda (needed for a lot of things)
+Writing code into a terminal is called using the command line. IHME has a helpful training 
+on the command line in Module 1 of the IHME Learn training `Computational Infrastructure Level 1 <https://ihme.brightspace.com/d2l/home/7028>`_. Command lines technically interact with an 
+operating system on your computer. 
 
+Operating systems are things like Windows or Mac that help a computer run and provide some basic 
```

Comment:
```suggestion
Operating systems are things like Windows or macOS that help a computer run and provide some basic 
```
Mac is the computer, the operating system that runs it is macOS.

Reply:
Thank you!